### PR TITLE
HTML & CSS Practice

### PR DESCRIPTION
**Pull request by yuryzaytsev**
### 5. HTML & CSS Practice

[Demo](https://yuryzaytsev.github.io/hooli-style-popup-example) |
[Code base](https://github.com/yuryzaytsev/hooli-style-popup-example)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by Temu4** [comment]
Hey! Looks very pretty! 😸  🚀 
It is not a review, just visual testing.

File: submissions/yuryzaytsev/index.html [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+		s0.512-0.098,0.707-0.293c0.391-0.391,0.391-1.023,0-1.414C38.75,26.771,33.279,24.508,27.454,24.508z"></path>
+	<path d="M27.454,33.916c-3.612,0-6.551,2.939-6.551,6.552s2.939,6.552,6.551,6.552c3.613,0,6.552-2.939,6.552-6.552
+		S31.067,33.916,27.454,33.916z M27.454,45.019c-2.51,0-4.551-2.042-4.551-4.552s2.042-4.552,4.551-4.552s4.552,2.042,4.552,4.552
+		S29.964,45.019,27.454,45.019z"></path>
+</g>
+</symbol></svg>
+  </div>
+
+  <header class="header container">
+  <ul class="header__list">
```
Comment (A-Ostrovnyy) [inline submissions/yuryzaytsev/index.html:303]: For navigation preferable(more semantic) markup `nav > ul > li`.

File: submissions/yuryzaytsev/css/style.css [hunk 2]
```diff
@@ -0,0 +1,685 @@
...
+.popup #btn-more:checked ~ .popup__btn {
+  display: none; }
+
+.popup #btn-more:checked ~ .popup__list--hidden {
+  display: grid; }
+
+.popup__list {
+  display: grid;
+  gap: 5px;
+  grid-template-columns: 90px 90px 90px;
```
Comment (A-Ostrovnyy) [inline submissions/yuryzaytsev/css/style.css:563]: In this case preferable to use `repeat()` function. 
[Read more about it](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat).

File: submissions/yuryzaytsev/css/style.css [hunk 3]
```diff
@@ -0,0 +1,685 @@
...
+  display: none; }
+
+/**
+ * Add the correct display in IE 10.
+ */
+[hidden] {
+  display: none; }
+
+/* Блоки */
+.container {
```
Comment (A-Ostrovnyy) [inline submissions/yuryzaytsev/css/style.css:386]: Use [Prettier](https://prettier.io/) for code formatting.

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Eye candy!
In PR of your home works, into current repository, add only  source code files, No need to add images, fonts, sourcemap files, or minified/ compiled files.

File: submissions/yuryzaytsev/index.html (hunk 1 above)
Comment (yuryzaytsev) [reply submissions/yuryzaytsev/index.html]: Fixed.

File: submissions/yuryzaytsev/css/style.css (hunk 2 above)
Comment (yuryzaytsev) [reply submissions/yuryzaytsev/css/style.css]: Fixed.

File: submissions/yuryzaytsev/css/style.css (hunk 3 above)
Comment (yuryzaytsev) [reply submissions/yuryzaytsev/css/style.css:51]: Ok, but what's wrong with the quoted line or it's about the whole formatting style? I tried to use Prettier in VS Code, but the result looked the same.

**Comment by yuryzaytsev** [comment]
### Thank you for review!
---
> In PR of your home works, into current repository, add only source code files, No need to add images, fonts, sourcemap files, or minified/ compiled files.

I deleted fonts, sourcemap and minified files, but don't touched inline SVG images. Is it Ok?

**Review by A-Ostrovnyy — APPROVED** [review]
LGTM
Well done

**Pull request merged**